include/zephyr/drivers/video-controls.h

```diff
@@ -81,9 +100,110 @@ enum video_power_line_frequency {
 	VIDEO_CID_POWER_LINE_FREQUENCY_AUTO = 3,
 };
 
-/** Balance of colors in direction of blue (cold) or red (warm) */
+/** Enables automatic hue control by the device.
+ * Setting @ref VIDEO_CID_HUE while automatic hue control is enabled is undefined.
+ * Drivers should ignore such request.
+ */
+#define VIDEO_CID_HUE_AUTO (VIDEO_CID_BASE + 25)
+
+/** White balance settings as a color temperature in Kelvin.
+ * A driver should have a minimum range of 2800 (incandescent) to 6500 (daylight).
+ */
 #define VIDEO_CID_WHITE_BALANCE_TEMPERATURE (VIDEO_CID_BASE + 26)
 
+/** Adjusts the sharpness filters in a camera.
+ * The minimum value disables the filters, higher values give a sharper picture.
+ */
+#define VIDEO_CID_SHARPNESS (VIDEO_CID_BASE + 27)
+
+/** Adjusts the backlight compensation in a camera.
+ * The minimum value disables backlight compensation.
+ */
+#define VIDEO_CID_BACKLIGHT_COMPENSATION (VIDEO_CID_BASE + 28)
+
+/** Chroma automatic gain control. */
+#define VIDEO_CID_CHROMA_AGC (VIDEO_CID_BASE + 29)
+
+/** Enable the color killer, i.e. force a black and white image in case of a weak video signal. */
+#define VIDEO_CID_COLOR_KILLER (VIDEO_CID_BASE + 30)
+
+/** Selects a color effect. */
+#define VIDEO_CID_COLORFX (VIDEO_CID_BASE + 31)
+enum video_colorfx {
+	VIDEO_COLORFX_NONE = 0,
+	VIDEO_COLORFX_BW = 1,
+	VIDEO_COLORFX_SEPIA = 2,
+	VIDEO_COLORFX_NEGATIVE = 3,
+	VIDEO_COLORFX_EMBOSS = 4,
+	VIDEO_COLORFX_SKETCH = 5,
+	VIDEO_COLORFX_SKY_BLUE = 6,
+	VIDEO_COLORFX_GRASS_GREEN = 7,
+	VIDEO_COLORFX_SKIN_WHITEN = 8,
+	VIDEO_COLORFX_VIVID = 9,
+	VIDEO_COLORFX_AQUA = 10,
+	VIDEO_COLORFX_ART_FREEZE = 11,
+	VIDEO_COLORFX_SILHOUETTE = 12,
+	VIDEO_COLORFX_SOLARIZATION = 13,
+	VIDEO_COLORFX_ANTIQUE = 14,
+	VIDEO_COLORFX_SET_CBCR = 15,
+	VIDEO_COLORFX_SET_RGB = 16,
+};
+
+/* Enable Automatic Brightness. */
+#define VIDEO_CID_AUTOBRIGHTNESS (VIDEO_CID_BASE + 32)
+
+/** Switch the band-stop filter of a camera sensor on or off, or specify its strength.
+ * Such band-stop filters can be used, for example, to filter out the fluorescent light component.
+ */
+#define VIDEO_CID_BAND_STOP_FILTER (VIDEO_CID_BASE + 33)
+
+/** Rotates the image by the specified angle.
+ * Common angles are 90, 270 and 180. Rotating the image to 90 and 270 will reverse the height and
+ * width of the display window.
+ */
+#define VIDEO_CID_ROTATE (VIDEO_CID_BASE + 34)
+
+/** Sets the background color on the current output device.
+ * The 32-bit of the supplied value are interpreted as follow:
+ * - bits [7:0] as Red component,
+ * - bits [15:8] as Green component,
+ * - bits [23:16] as Blue component,
+ * - bits [31:24] must be zero.
```

Comment:
Note: this mapping between bits and color channels is the opposite for this control, as all others are Blue, Green, Red.